src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -709,6 +709,13 @@ public static void ThrowNotSupportedException_NoMetadataForType(Type type, IJson
             throw new NotSupportedException(SR.Format(SR.NoMetadataForType, type, resolver?.GetType().FullName ?? "<null>"));
         }
 
+
+        [DoesNotReturn]
+        public static void ThrowNotSupportedException_ConstructorContainsNullParameterNames(Type declaringType)
```

Comment:
Who calls this new method? I don't see any callsite for it. Is line 246 of ReflectionJsonTypeInfoOfT.cs supposed to be calling it?

Reply:
Good catch, yes. Not sure why I ended up using the wrong throw helper. I'll file a follow-up PR.